packages/server/src/integrations/rest.ts

```diff
@@ -131,7 +131,10 @@ class RestIntegration implements IntegrationBase {
     let data, raw, headers
     const contentType = response.headers.get("content-type") || ""
     try {
-      if (contentType.includes("application/json")) {
+      if (response.status === 204) {
+        data = []
```

Comment:
Why an empty array instead of just undefined?

Reply:
I was concerned about null pointers, but I can see that it was already handled:
https://github.com/Budibase/budibase/blob/895c9a9ac36532371a3f97327678099887588613/packages/frontend-core/src/fetch/QueryFetch.js#L78

Think I'll leave it as an empty array for good measure though!